content/en/user-manual/graphics/layers/index.md

```diff
@@ -50,8 +50,8 @@ PlayCanvas applications are created with a set of default layers. You should lea
 
 ![Default Layers][1]
 
-1. **World (Opaque)** - Used to render components that are not transparent and most opaque component meshes.
-1. **Depth (Opaque)** - The depth layer is for advanced users only.
+1. **World (Opaque)** - Used to render components that are not transparent and most opaque component meshes.
+1. **Depth (Opaque)** - Used to capture the color or the depth buffer of the scene, see [Depth Layer][depth-layer].
```

Comment:
Minor: Keep the numbering system for links if the page already uses them or change all of them so we don't mix and match please

Reply:
I think it's a lot more convenient to move to a name based, even one by one? Does it create some problems?

Reply:
I think I'm with @yaustar on this - it's pretty messy to mix the two styles. So I'd recommend switching to `7` please.